src/resources/Catalogs/ProductListingConfigurationInterfaces.ts

```diff
@@ -42,4 +46,6 @@ export enum RankingTypeEnum {
     BOOST = 'BOOST',
     BURY = 'BURY',
     PIN = 'PIN',
+    INCLUDE = 'INCLUDE',
```

Comment:
If I remember correctly, `include` and `exclude` configurations won't be stored in the RankingConfiguration table because it doesn't affect `ranking` but searchable products.

Reply:
What type are they then ?

Reply:
A type that doesn't exist yet, I think it'll be something along the line of SearchTypeEnum, but it's best to remove these for now and we'll add them when the new objects will be defined.